extensions/linz/json-schema/schema.json

```diff
@@ -19,8 +19,50 @@
               "minLength": 1
             }
           }
+        },
+        {
+          "$ref": "#/definitions/security_classification"
+        },
+        {
+          "$ref": "#/definitions/stac_extensions"
         }
       ]
     }
-  ]
+  ],
+  "definitions": {
+    "stac_extensions": {
+      "type": "object",
+      "required": ["stac_extensions"],
+      "properties": {
+        "stac_extensions": {
+          "type": "array",
+          "contains": {
+            "const": "https://linz.github.io/STAC/extensions/linz/v1.0.0/schema.json"
+          }
+        }
+      }
+    },
+    "security_classification": {
+      "type": "object",
+      "required": ["linz:security_classification"],
+      "properties": {
+        "linz:security_classification": {
+          "type": "string",
+          "enum": [
+            "Unclassified",
+            "IN-CONFIDENCE",
+            "SENSITIVE",
+            "RESTRICTED",
+            "CONFIDENTIAL",
+            "SECRET",
+            "TOP-SECRET"
+          ]
+        }
+      },
+      "patternProperties": {
+        "^(?!linz:)": {}
```

Comment:
This subschema is empty. Is that intentional?

Reply:
I was copying the format in https://github.com/linz/STAC/blob/49aaf21177ba8f932967ab0e6beeeed1237967f2/extensions/camera/json-schema/schema.json#L62-L65

If it is removed then it cause a schema validation error because `additionalProperties` is `false`. see this [example](https://www.jsonschemavalidator.net/s/hKcm0s7m)

Although not sure if we should allow additional properties with the `linz:` schema or not. Maybe should? i.e. change [this](https://www.jsonschemavalidator.net/s/HEwxurVo) to [this](https://www.jsonschemavalidator.net/s/NybtKj1a)

Reply:
Looks like the standard way to do it in other extensions tho.

https://github.com/stac-extensions/scientific/blob/cff9ff73e2502391102d8e71553e01ec15be3784/json-schema/schema.json#L193-L198

https://github.com/stac-extensions/projection/blob/7f1d1c9db7c33dba0944972a983d74fd480d4c62/json-schema/schema.json#L189-L194

Whether is good json schema or not... 🤷

Reply:
I can't claim to understand this idiom, but the reference to third parties is compelling. Let's just leave this as is.